lib/slave/cracker.go

```diff
@@ -8,14 +8,20 @@ import (
 	"github.com/aaronang/cong-the-ripper/lib/slave/hasher"
 )
 
-func Execute(task lib.Task, successChan chan CrackerSuccess, failChan chan CrackerFail) {
-	bd := brutedict.New(&task)
+func Execute(task *task, successChan chan CrackerSuccess, failChan chan CrackerFail) {
+	bd := brutedict.New(&task.Task)
```

Comment:
Nope not possible. Signature of `brutedict.New` is `func New(task *lib.Task) (bd *BruteDict)`. Therefore a pointer of a `Task` needs to be given as parameter, not `task`.